printhelp-module.js

```diff
@@ -0,0 +1,26 @@
+var clc = require('cli-color');
+var charm = require('charm')();
+
+var methods = {};
+
+methods.update_start_inspect = function (start_inspect) {
+  return start_inspect + 8;
+};
+
+methods.update_start_solve = function (start_solve) {
+  return start_solve + 8;
+};
+
+methods.print_help  = function (start_inspect) {
+  console.log("Keyboard shortcuts:");
```

Comment:
This function should print some lines on the right column and some on the left. See [this](https://github.com/icyflame/cli-cube-timer/blob/83ae23472c31b1fb6143acd17804719e621b5c5e/index.js#L377-L390)

Reply:
Sir, should I leave the new js file and call function from the js whenever there is need to print the help message?
`This function should print some lines on the right column and some on the left.` can you please elaborate it as I am little confused?

Reply:
2. If you see this block of code: https://github.com/icyflame/cli-cube-timer/blob/83ae23472c31b1fb6143acd17804719e621b5c5e/index.js#L377-L390

You will notice that the cursor is repositioned using `charm` before printing using `console.log`. So, the `print_help` function must also do this.